### DISCUSSION TITLE
Read status on start problem after KNX device node restart

### DISCUSSION THREAD

**Discussion by jomifa** [Supergiovane/node-red-contrib-knx-ultimate#412] [General]
**Salutation (i'll not respond to users that doesn’t wrote at least "Hello")**

**Describe the bug**
Hello,
I have a question about the specification in a KNX device node.

In the section: “Read status on start” you can specify “Restart last value saved to file prior to disconnection. If the file is missed read from KNX Bus ....”. I assume that the file that is described is an internal file and does not need to be specified by the user. I have observed that when the node is restarted, an attempt is made to read from the KNX bus and the file is not evaluated. Are there problems in this area?

Greetings
Joachim

**To Reproduce**
Write here the steps to reproduce the behavior, for example:

**Expected behavior**
Write here a clear and concise description of what you expected to happen.
After a restart of the KNX device node, the last value for KNX should be provided. A KNX read telegram should only be sent if the file is not available.
**Screenshots**
Here, if applicable, add screenshots to help explain your problem.
![grafik](https://github.com/user-attachments/assets/0e04d0b4-e1f2-4f56-8ed8-b35344f7c9b2)

**Knx-Ultimate Version**
3.3.8

**Are you running node-red behind homematic, docker or anything similar?**
Node-Red running alone in RaspberryPi

**Additional context**

**Comment by Supergiovane** [comment]
Hi
The file is internal and is a json with the last node value. 
Everytime you enter the configuration window, the file is deleted, so it doesn’t find any file and it reads from the bus. 
Try leaving it running without entering the node config.

**Comment by jomifa** [comment]
Hello,
I have not changed the “node config”.

I just saw that another discussion post (#409) describes the same problem.

It is also strange how the display of the KNX device node behaves

1. normal status of the KNX device node

![Screenshot 2024-12-29 134203](https://github.com/user-attachments/assets/b0ddeddc-2ac8-4d1a-8004-79b0369d1fa4)


2. query of the GA from the ETS or the Gira Home Server. The result of the query is correct, the display in the KNX Device in Node Red suggests that a 0 is returned to the caller. A short time later, the KNX device node displays the correct value again.

![Screenshot 2024-12-29 134316](https://github.com/user-attachments/assets/afe1ec77-24db-496f-858a-ec34bfe751a6)

![Screenshot 2024-12-29 134523](https://github.com/user-attachments/assets/e73e02ca-8885-47ce-928a-dc11578ad615)
